hokusai/staging.yml

```diff
@@ -93,6 +93,17 @@ spec:
                 operator: In
                 values:
                 - foreground
+                - foreground-spot
+      topologySpreadConstraints:
+      - maxSkew: 1
+        topologyKey: tier
+        whenUnsatisfiable: ScheduleAnyway
```

Comment:
if spreading is not achievable, for example, if there are no instance available in `foreground-spot`, just schedule everything in `foreground`.